qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -187,6 +187,13 @@ class DefaultPoliciesTest extends BaseSpecification {
             log.info "Temporarily enabled policy '${policyName}'"
             policyEnabled = true
         }
+        // debugging to see if the test fails due to incomplete scan
+        if (policyName == "Apache Struts: CVE-2017-5638") {
+            def image = ImageService.scanImage("library/nginx:1.10", true)
+            if (!hasApacheStrutsVuln(image)) {
+                log.warn("[Apache struts] CVE-2017-5638 is absent from image scan")
```

Comment:
IMO you should add the JIRA to the log messages to make grepping for it in the OSCI logs much easier.